### PR TITLE
blogpost for hadoop summit

### PR DESCRIPTION
**Pull request by rmetzger**

### CODE REVIEW

File: _posts/2014-01-10-stratosphere-hadoop-summit.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title:  'Stratosphere got accepted to the Hadoop Summit Europe in Amsterdam'
+date:   2014-01-10 10:57:18
+categories: news blog
+---
+
+
+The Stratosphere team is proud to announce that it is going to present at the [Hadoop Summit 2014 in Amsterdam](http://hadoopsummit.org/amsterdam/) in April 2-3 with our talk "Big Data looks tiny from Stratosphere" on the "Future of Hadoop Track". Stratosphere already made it into the top 5 in the [Community Vote](https://hadoopsummit.uservoice.com/forums/196822-future-of-apache-hadoop/filters/top) that took place by the end of last year.
```
Comment (uce) [inline _posts/2014-01-10-stratosphere-hadoop-summit.md]: "Future of Hadoop" Track not "Future of Hadoop Track", right?

**Comment by uce** [comment]
Brief and to the point. I like it!

**Comment by StephanEwen** [comment]
Nice!

Please address Ufuk's comment as "Future of Hadoop" track and merge.

**Comment by StephanEwen** [comment]
Version with a few corrections and shorter sentences.

The Stratosphere team is proud to announce that it is going to present at the [Hadoop Summit 2014 in Amsterdam](http://hadoopsummit.org/amsterdam/) on April 2-3. Our talk "Big Data looks tiny from Stratosphere" is part of the "Future of Hadoop" track. The talk abstract already made it into the top 5 in the [Community Vote](https://hadoopsummit.uservoice.com/forums/196822-future-of-apache-hadoop/filters/top) that took place by the end of last year.

**Pull request merged**